uwuno/src/main/java/com/learning/uwuno/room.java

```diff
@@ -0,0 +1,21 @@
+package com.learning.uwuno;
+
+import java.util.List;
+
+public class room {
+    public room(int id, String roomName) {
+        this.uid = id;
+        this.roomName = roomName;
+    }
+
+    // Class Variables
```

Comment:
Not sure, but I think it's best practice to put class members first (before all the constructor/methods etc.). So put it right after line 5 "public class room {..."